components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -863,14 +886,15 @@ private void updateBasicApplicationData(ServiceProvider serviceProvider, Connect
             storeAppPrepStmt.setString(1, applicationName);
             storeAppPrepStmt.setString(2, description);
             storeAppPrepStmt.setString(3, isSaasApp ? "1" : "0");
+            storeAppPrepStmt.setString(4, isDiscoverable ? "1" : "0");
```

Comment:
Shall we use named prepared statement here if possible?
When we implement update by resourceId methods, it would be much easier to update the code here.

Reply:
Fixed